src/class_objectProcessor.py

```diff
@@ -471,6 +473,8 @@ def processmsg(self, data):
             try:
                 # continue decryption attempts to avoid timing attacks
                 if initialDecryptionSuccessful:
+                    if BMConnectionPool().trustedPeer:
```

Comment:
Interesting.

Reply:
I can drop e002984. It's made in the separate commit intentionally. But then I should probably change the docstring from previous commit. Because if it will appear in the doc, it may be interpreted as promise.